docs/algorithms/apply_gate_library.rst

```diff
@@ -9,4 +9,10 @@ Applies an FCN :ref:`gate library <fcn-gate-libraries>` to a ``gate_level_layout
 in the same :ref:`technology <fcn-cell-technologies>` as the provided gate library. Thereby, this function creates cell-accurate
 implementations for each gate present in the passed ``gate_level_layout``.
 
-.. doxygenfunction:: fiction::apply_gate_library(const GateLyt& lyt)
+.. tabs::
+    .. tab:: C++
```

Comment:
Would it be possible to use `.. code-tab:: c++` and `.. code-tab:: py` everywhere to have the tabs sync and render code automatically? Or would that break the visuals somewhere?

Reply:
Unfortunately, `doxygenfunction` and `auto-doc` don't work with code-tabs.

Reply:
No problem. Many thanks for trying!